frontend/resources/locales.json

```diff
@@ -3823,166 +3834,172 @@
     "translations" : {
       "en" : "Paste",
       "es" : "Pegar"
     }
   },
   "workspace.shape.menu.reset-overrides" : {
-    "used-in" : [ "src/app/main/ui/workspace/sidebar/options/component.cljs:80", "src/app/main/ui/workspace/sidebar/options/component.cljs:85", "src/app/main/ui/workspace/context_menu.cljs:165", "src/app/main/ui/workspace/context_menu.cljs:175" ],
+    "used-in" : [ "src/app/main/ui/workspace/context_menu.cljs:182", "src/app/main/ui/workspace/context_menu.cljs:192", "src/app/main/ui/workspace/sidebar/options/component.cljs:96", "src/app/main/ui/workspace/sidebar/options/component.cljs:101" ],
     "translations" : {
       "en" : "Reset overrides",
       "es" : "Deshacer modificaciones"
     }
   },
   "workspace.shape.menu.show" : {
-    "used-in" : [ "src/app/main/ui/workspace/context_menu.cljs:135" ],
+    "used-in" : [ "src/app/main/ui/workspace/context_menu.cljs:152" ],
     "translations" : {
       "en" : "Show",
       "es" : "Mostrar"
     }
   },
   "workspace.shape.menu.show-master" : {
-    "used-in" : [ "src/app/main/ui/workspace/sidebar/options/component.cljs:82", "src/app/main/ui/workspace/context_menu.cljs:169" ],
+    "used-in" : [ "src/app/main/ui/workspace/context_menu.cljs:186", "src/app/main/ui/workspace/sidebar/options/component.cljs:98" ],
     "translations" : {
       "en" : "Show master component",
       "es" : "Ver componente maestro"
     }
   },
   "workspace.shape.menu.ungroup" : {
-    "used-in" : [ "src/app/main/ui/workspace/context_menu.cljs:123" ],
+    "used-in" : [ "src/app/main/ui/workspace/context_menu.cljs:140" ],
     "translations" : {
       "en" : "Ungroup",
       "es" : "Desagrupar"
     }
   },
   "workspace.shape.menu.unlock" : {
-    "used-in" : [ "src/app/main/ui/workspace/context_menu.cljs:141" ],
+    "used-in" : [ "src/app/main/ui/workspace/context_menu.cljs:158" ],
     "translations" : {
       "en" : "Unlock",
       "es" : "Desbloquear"
     }
   },
   "workspace.shape.menu.unmask" : {
-    "used-in" : [ "src/app/main/ui/workspace/context_menu.cljs:127" ],
+    "used-in" : [ "src/app/main/ui/workspace/context_menu.cljs:144" ],
     "translations" : {
       "en" : "Unmask",
       "es" : "Quitar máscara"
     }
   },
   "workspace.shape.menu.update-master" : {
-    "used-in" : [ "src/app/main/ui/workspace/sidebar/options/component.cljs:81", "src/app/main/ui/workspace/context_menu.cljs:167" ],
+    "used-in" : [ "src/app/main/ui/workspace/context_menu.cljs:184", "src/app/main/ui/workspace/context_menu.cljs:196", "src/app/main/ui/workspace/sidebar/options/component.cljs:97", "src/app/main/ui/workspace/sidebar/options/component.cljs:103" ],
     "translations" : {
       "en" : "Update master component",
       "es" : "Actualizar componente maestro"
     }
   },
+  "workspace.sidebar.history" : {
+    "used-in" : [ "src/app/main/ui/workspace/left_toolbar.cljs:122" ],
+    "translations" : {
+      "en" : "History (%s)"
```

Comment:
I think for new texts we should at least give the spanish translation, which is already complete.